bindings/go/plugin/internal/testplugin/main.go

```diff
@@ -110,6 +115,11 @@ func main() {
 
 	logger.Info("registered test plugin")
 
+	capabilities.AddConfigType(runtime.Type{
```

Comment:
this type should be derived from the config above right? now you configure it manually

Reply:
>this type should be derived from the config above right?

Hm? Which one would that be?

Reply:
I was hoping you would have a scheme that you derive the type from but for this test its fine :)